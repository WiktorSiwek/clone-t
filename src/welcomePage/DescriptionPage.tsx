import { faComment, faSearch, faUsers } from '@fortawesome/free-solid-svg-icons';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
  color: white;
  line-height: 400%;
  margin: auto;
  width: 400px;
  padding: 30vh;
  display: flex;
  flex-direction: row;
`;

const Column = styled.div`
  display: flex;
  flex-direction: column;
`;

const Text = styled.span`
  font-size: 17px;
  font-weight: bold;
  padding-left: 15px;
`;

const Icon = styled.span`
  font-size: 28px;
`;


export default function DescriptionPage() {
  return (
    <Container>
      <Column>
        <Icon>
          <FontAwesomeIcon icon={faSearch} />
        </Icon>
        <Icon>
          <FontAwesomeIcon icon={faUsers} />
        </Icon>
        <Icon>
          <FontAwesomeIcon icon={faComment} />
        </Icon>
      </Column>
      <Column>
        <Text>Obserwuj to, co Cię interesuje</Text>
        <Text>Dowiedz się, o czym rozmawiają inni.</Text>
        <Text>Dołącz do rozmowy.</Text>
      </Column>
    </Container>
  )
}

