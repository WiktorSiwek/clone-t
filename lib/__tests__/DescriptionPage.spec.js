import React from 'react';
import TestRenderer from 'react-test-renderer';
import DescriptionPage from '../components/welcomePage/DescriptionPage';

describe('Description Page', () => {
   it('renders as expected', () => {
      const tree = TestRenderer.create(<DescriptionPage />);

      expect(tree).toMatchSnapshot();
   });
});