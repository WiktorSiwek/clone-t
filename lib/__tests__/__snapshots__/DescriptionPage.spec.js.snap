// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description Page renders as expected 1`] = `
<div
  className="sc-bdVaJa tRdKp"
>
  <div
    className="sc-bwzfXH kulSxP"
  >
    <span
      className="sc-bxivhb hsrDiO"
    >
      <i
        className="Icon Icon--search"
      />
    </span>
    <span
      className="sc-bxivhb hsrDiO"
    >
      <i
        className="Icon Icon--people"
      />
    </span>
    <span
      className="sc-bxivhb hsrDiO"
    >
      <i
        className="Icon Icon--reply"
      />
    </span>
  </div>
  <div
    className="sc-bwzfXH kulSxP"
  >
    <span
      className="sc-htpNat jwtfWe"
    >
      Follow your interests.
    </span>
    <span
      className="sc-htpNat jwtfWe"
    >
      Hear what people are talking about.
    </span>
    <span
      className="sc-htpNat jwtfWe"
    >
      Join the conversation.
    </span>
  </div>
</div>
`;
